### PR TITLE
Project Beats: Add 'run' word to run button

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Adds "Run" and "Stop" to the run/stop button, and changes its presentation to look more like other run buttons on the site, as per product/design guidance. See [slack thread](https://codedotorg.slack.com/archives/C04TH8400AU/p1682381372779379) for further details.

Updated with reduced font size and padding:

Screenshot 2023-05-11 at 10 36 55 AM

Also reworked the Controls component a little bit to make all the icons into `<button>` elements for better accessibility.

## Links

https://codedotorg.atlassian.net/browse/SL-753

## Testing story

Tested locally.

### CODE REVIEW

File: apps/src/music/views/Controls.jsx [hunk 1]
```diff
@@ -59,56 +60,83 @@ const Controls = ({
...
-          className={moduleStyles.iconButton}
-        />
+          type="button"
+        >
+          <FontAwesome
+            icon={isPlaying ? 'stop' : 'play'}
+            className={moduleStyles.playStopIcon}
+          />
+          <div className={moduleStyles.text}>
+            {isPlaying ? commonI18n.stop() : commonI18n.runProgram()}
```
Comment (mikeharv) [inline apps/src/music/views/Controls.jsx:118]: If the goal is to align this button to other labs, should the label here be "reset" instead of "stop"?

Comment (sanchitmalhotra126) [reply apps/src/music/views/Controls.jsx:118]: Yeah, wasn't sure about this. 'Reset' felt weird to me from a music playback perspective, but maybe it's not that confusing? Let me check with @moneppo

File: apps/src/music/views/Controls.jsx [hunk 2]
```diff
@@ -59,56 +60,83 @@ const Controls = ({
...
-      <FontAwesome
-        icon={icon}
-        className={moduleStyles.iconButton}
-        onClick={onClick}
-      />
-    </div>
+  const renderIconButton = (icon, onClick, hide) => (
+    <button
+      className={classNames(
+        moduleStyles.controlButton,
```
Comment (breville) [inline apps/src/music/views/Controls.jsx:66]: The button's default `font-size` of `large` (which maps to `18px`) feels a bit jumbo compared to everything else on the page right now.  I'd be inclined to use `14px` for now.  And then also reduce the icon to suit that size.

**Comment by breville** [comment]
It looks like there is a broader button issue that we should resolve before merging: 

https://github.com/code-dot-org/code-dot-org/assets/2205926/88b9320c-33da-4608-a26a-081e349d5d4c

When pressing the button, its height changes.  As well as not looking quite correct, it has an impact on surrounding UI.

**Comment by sanchitmalhotra126** [comment]
> When pressing the button, its height changes. As well as not looking quite correct, it has an impact on surrounding UI.

Good call, this is strange, it seems like the border is disappearing. I'll take a look.

Edit: this doesn't seem to be happening on my end? 

https://github.com/code-dot-org/code-dot-org/assets/85528507/5479bb82-e192-4d8a-b08b-06560544db93

**Comment by breville** [comment]
> Edit: this doesn't seem to be happening on my end?

It looks like the button styling was fixed externally, and it now has a dark grey border when being pressed, preventing the size change.

**Review by breville — APPROVED** [review]

**Comment by breville** [comment]
> > Edit: this doesn't seem to be happening on my end?
> 
> It looks like the button styling was fixed externally, and it now has a dark grey border when being pressed, preventing the size change.

Oh, turns out the missing border comes from [here](https://github.com/code-dot-org/code-dot-org/blob/fcd401860e968b5dfbd184c54e3f28c8a706e247/apps/style/music/style.scss#L9), and only happens at the `/levels` and `/s/allthethings/` URLs, but not at `/projectbeats`.

**Comment by breville** [comment]
> > > Edit: this doesn't seem to be happening on my end?
> > 
> > 
> > It looks like the button styling was fixed externally, and it now has a dark grey border when being pressed, preventing the size change.
> 
> Oh, turns out the missing border comes from [here](https://github.com/code-dot-org/code-dot-org/blob/fcd401860e968b5dfbd184c54e3f28c8a706e247/apps/style/music/style.scss#L9), and only happens at the `/levels` and `/s/allthethings/` URLs, but not at `/projectbeats`.

Fix prepped at https://github.com/code-dot-org/code-dot-org/pull/51936.

**Comment by sanchitmalhotra126** [comment]
> Fix prepped at #51936.

Thanks for the quick fix!

**Pull request merged**